presto-main/src/main/java/com/facebook/presto/testing/MaterializedResult.java

```diff
@@ -383,30 +383,6 @@ else if (prestoValue instanceof SqlDecimal) {
         return new MaterializedRow(prestoRow.getPrecision(), convertedValues);
     }
 
-    public MaterializedResult toTimeZone(DateTimeZone oldTimeZone, DateTimeZone newTimeZone)
```

Comment:
Dear Reviewer,

The only usage of this was in `H2QueryRunner`, labeled with _"H2 produces dates in the JVM time zone instead of the session timezone"_. -- actually the H2 driver behavior was not to be blamed... It just emphasizes the need for dumping any usage of `java.sql.Date`.